src/Sylius/Behat/Resources/config/services/contexts/ui.xml

```diff
@@ -293,6 +293,9 @@
 
         <service id="sylius.behat.context.ui.shop.product" class="%sylius.behat.context.ui.shop.product.class%" scope="scenario">
             <argument type="service" id="sylius.behat.page.shop.product.show" />
+            <argument type="service" id="sylius.behat.page.shop.product.index" />
```

Comment:
According to my comments above, this page should be gone. If we ever add list of all products to Sylius (without taxon filtering) then this would be a correct page to handle this.